Cargo.toml

```diff
@@ -28,7 +28,8 @@ members = [
 # > explicit about the dependency relationships between the crates.
 [patch.crates-io]
 # Oak.
-asylo_rust = { path = "oak/asylo_rust" }
+eal = { path = "oak/server/rust/eal" }
```

Comment:
Not sure if it's too late for this, but I'm not a big fan of EAL, especially since in some cases the "enclave" may be a VM, or there may be no enclave at all (e.g. dev server).

How about `oak_platform` and `oak_platform_asylo`, or something like that?

Reply:
I'm happy with whatever name we go with, but it makes sense to decide before this PR. @daviddrysdale do you have a preference over `oak_platform` vs `eal` vs something else?

Reply:
Other alternatives:

- oak_env (similar to https://doc.rust-lang.org/reference/conditional-compilation.html#target_env)
- oak_sys

Reply:
cc @project-oak/core if anyone has any suggestion / preference.

Reply:
Done, switched to oak_platform.